extension/chrome/elements/compose.ts

```diff
@@ -31,17 +31,20 @@ Catch.try(async () => {
 
   const ksLookupsByEmail: { [key: string]: PubkeySearchResult | Contact } = {};
 
-  const uncheckedUrlParams = Env.urlParams(['acctEmail', 'parentTabId', 'draftId', 'placement', 'frameId', 'threadId', 'skipClickPrompt', 'ignoreDraft', 'debug']);
+  const uncheckedUrlParams = Env.urlParams(['acctEmail', 'parentTabId', 'draftId', 'placement', 'frameId', 'threadId',
+    'threadMsgId', 'skipClickPrompt', 'ignoreDraft', 'debug', 'removeAfterClose']);
   const acctEmail = Assert.urlParamRequire.string(uncheckedUrlParams, 'acctEmail');
   const parentTabId = Assert.urlParamRequire.string(uncheckedUrlParams, 'parentTabId');
   const frameId = Assert.urlParamRequire.string(uncheckedUrlParams, 'frameId');
   const skipClickPrompt = uncheckedUrlParams.skipClickPrompt === true;
   const ignoreDraft = uncheckedUrlParams.ignoreDraft === true;
+  const removeAfterClose = uncheckedUrlParams.removeAfterClose === true;
   const placement = Assert.urlParamRequire.oneof(uncheckedUrlParams, 'placement', ['settings', 'gmail', undefined]);
   const disableDraftSaving = false;
   const debug = uncheckedUrlParams.debug === true;
   let draftId = Assert.urlParamRequire.optionalString(uncheckedUrlParams, 'draftId') || '';
   let threadId = Assert.urlParamRequire.optionalString(uncheckedUrlParams, 'threadId') || '';
+  const threadMsgId = Assert.urlParamRequire.optionalString(uncheckedUrlParams, 'threadMsgId') || '';
```

Comment:
We should call this `replyToMsgId` for clarity, since that's how we're using it now

Reply:
That means maybe we don't need `threadId`? Since we have to make an api call anyway? (to retrieve message info - in the same api call we can retrieve `threadId` instead of getting it through url params?)

Reply:
Let me check, if we don't need it anymore I will remove it

Reply:
Look what I found here:
```TS
  reinsertReplyBox = (subject: string, threadId: string) => {
    const params: FactoryReplyParams = { subject, sendAs: this.sendAs, replyMsgId: threadId };
    $('.reply_message_iframe_container:visible').last().append(this.factory.embeddedReply(params, false, true)); // xss-safe-value
  }
```
Is it correct to pass threadId into replyMsgId; I'm not sure

Reply:
Sort of, the first message in the thread has the same `msgId` as the `threadId`. But we should not rely on that, and properly fix it.